newIDE/app/src/Export/ExportLauncher.js

```diff
@@ -70,11 +75,35 @@ export default class ExportLauncher extends Component<Props, State> {
     ),
   };
   buildsWatcher = new BuildsWatcher();
+  launchWholeExport: () => Promise<void>;
 
   componentWillUnmount() {
     this.buildsWatcher.stop();
   }
 
+  constructor(props: Props) {
+    super(props);
+    this._setupAchievementHook();
+  }
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    this._setupAchievementHook();
```

Comment:
Definitely React hooks are a blessing compared to "hacking" a component functions 😬

Reply:
Oh yes, completely!